salt/modules/virt.py

```diff
@@ -600,6 +607,8 @@ def _gen_xml(name,
     else:
         context['boot_dev'] = ['hd']
 
+    if 'loader' in kwargs:
+        context['loader'] = kwargs['loader']
```

Comment:
There are enough ``**kwargs`` to be removed in that module, don't add yet another one. Add a normal parameter with default value instead.

Reply:
Check, fixed this.